test/test.js

```diff
@@ -119,4 +119,24 @@ describe('postcss-increase-specificity', function() {
 	it('should not change the descendant rules of @keyframes', function() {
 		return testPlugin('./test/fixtures/keyframes.css', './test/fixtures/keyframes.expected.css');
 	});
+
+	it('should support disabled blocks of rules', function() {
+		return testPlugin('./test/fixtures/disabled-block.css', './test/fixtures/disabled-block.expected.css');
+	});
+
+	it('should support disabled blocks of rules with comments in between', function() {
+		return testPlugin('./test/fixtures/disabled-block-extra-comments.css', './test/fixtures/disabled-block-extra-comments.expected.css');
+	});
+
+	it('should support multiple disabled blocks of rules', function() {
+		return testPlugin('./test/fixtures/multiple-disabled-blocks.css', './test/fixtures/multiple-disabled-blocks.expected.css');
+	});
+
+	it('should support an entirely disabled stylesheet', function() {
+		return testPlugin('./test/fixtures/disabled-stylesheet.css', './test/fixtures/disabled-stylesheet.expected.css');
+	});
+
+	it('should support a partially disabled stylesheet', function() {
+		return testPlugin('./test/fixtures/partially-disabled-stylesheet.css', './test/fixtures/partially-disabled-stylesheet.expected.css');
+	});
```

Comment:
Let's add some tests for comments inside nested rules

```css
.foo {
	background: #f00;

	/* postcss-increase-specificity disable */
	.nested-foo {
		color: #00a;
	}
	/* postcss-increase-specificity enable */
}

.bar {
	color: #ef0;
}
```

Also do we expect this to disable the outer rules 🤷‍♀️? Let's add a test for this but I'm up for this case disable everything below regardless of where it's nested.

```css
.foo {
	background: #f00;

	/* postcss-increase-specificity disable */
	.nested-foo {
		color: #00a;
	}
}

.bar {
	color: #ef0;
}
```

Reply:
@MadLittleMods I thought at the `postcss` level we are dealing with CSS and not Sass?

Reply:
@track0x1 We are dealing with whatever the PostCSS parser can handle

Reply:
I think this request is out of scope for this PR as currently the plugin doesn't handle nested rules.

Reply:
Let's add a comment in the readme about this known limitation/design-decision.

> Currently, these disable/enable comments can not be nested inside a rule. Only comments in the global scope will be taken into account.